src/main/resources/application.yml

```
@@ -19,6 +21,15 @@ app:
     allowed-origins: http://localhost:3000,http://localhost:5173
 
+toss:
+  payments:
+    secret-key: ${TOSS_PAYMENTS_SECRET_KEY:test-payment-secret}
+    confirm-url: https://api.tosspayments.com/v1/payments/confirm
+    cancel-url: https://api.tosspayments.com/v1/payments/{paymentKey}/cancel
+
+springdoc:
+  api-docs:
+    path: /v3/api-docs
 springdoc:
```

YAMLlint (1.38.0)

[error] duplication of key "springdoc" in mapping

(key-duplicates)